spec/numbers_and_words/integer/fixture_examples/fr.yml

```diff
@@ -30,6 +39,8 @@ to_words:
     5000: cinq mille
     11000: onze mille
     21000: vingt et un mille
+    80000: quatre-vingt mille
+    200000: deux cent mille
```

Comment:
Even more confusing than 200 is written "deux cents" :laughing:

Reply:
Penible :)